storage/innobase/os/os0file.cc

```diff
@@ -10197,6 +10201,13 @@ Encryption::encrypt_log_block(
 	/* Set the encrypted flag. */
 	log_block_set_encrypt_bit(dst_ptr, true);
 
+	if(m_type == Encryption::KEYRING) {
+		const ulint crc = log_block_calc_checksum_crc32(dst_ptr);
+		log_block_set_checksum(dst_ptr, crc + m_key_version);
+		fprintf(stderr, "Encrypted with version %u\n", m_key_version);
```

Comment:
Huh?

Reply:
Uh, sorry about this and the formatting issues - I removed this and fixed most formatting problems based on the github diff, but looks like I forgot to push the changes or simply didn't add force...